apps/expo/src/components/Collections.tsx

```diff
@@ -39,7 +39,7 @@ export const CollectionsTab: FC<CollectionProps> = ({ tabData }) => {
         estimatedItemSize={5}
         showsVerticalScrollIndicator={false}
         renderItem={({ item }) => (
-          <View className=" mx-1 my-5">
+          <View className=" mx-4 my-5">
```

Comment:
Should have made a prop instead of directly modifying style. Chances are this could affect also the styling in library screen